chain/chain/src/store.rs

```diff
@@ -2037,6 +2066,14 @@ impl<'a> ChainStoreUpdate<'a> {
                     }
                 }
                 self.clear_chunk_data(min_chunk_height)?;
+                self.remove_state_dl_info(block_hash);
+                // Check Epoch switch
+                let cur_epoch_id = block.header().epoch_id();
+                if cur_epoch_id != next_epoch_id {
+                    // Current Block is the last one who needs current Epoch
+                    self.gc_col(ColEpochInfo, &cur_epoch_id.as_ref().into());
+                    self.gc_col(ColEpochStart, &cur_epoch_id.as_ref().into());
+                }
```

Comment:
Same here, let's defer this until after phase 1 (alternatively send this and the above as a separate PR against `post-phase-1` branch)